dist/src/main/resources/configuration/hawkular-realm-for-dev.json

```diff
@@ -86,6 +86,8 @@
       "enabled": true,
       "bearerOnly" : false,
       "publicClient": false,
+      "redirectUris": ["/*", "http://localhost:2772/*"],
+      "webOrigins": ["http://localhost:2772"],
```

Comment:
Btw. there is also https://issues.jboss.org/browse/HAWKULAR-372 I guess this is the right place to add also following origins: http://pastebin.test.redhat.com/292036, wdyt? I can create a PR for it not to complicate things and solve them separately.